motoman_gp200r_support/package.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp200r_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman GP200r (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman GP200r series of manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>GP200R - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information 
+      found in the online documentation files for the robot (GP200R_2D.zip
+      and GP200R_3D.zip from
+      https://www.cz.yaskawa.eu.com/en/products/robotics/motoman-robots/productdetail/product/gp200R/)
+      All urdfs are based on the default motion and joint velocity limits, 
+      unless noted otherwise.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author>Jakub Marsik</author>
+  <maintainer email="jakub@km-robotics.cz">Jakub Marsik (KM Robotics)</maintainer>
```

Comment:
Is this copy-pasta from the GP180 package?

Reply:
I have updated it to refer to myself and my manager as maintainers.